cpp/tests/transform/row_conversion.cpp

```diff
@@ -934,6 +935,8 @@ TEST_F(RowToColumnTests, BigStrings)
 
 TEST_F(RowToColumnTests, ManyStrings)
 {
+  // this test is very sensitive, so seed the random number generator
```

Comment:
Sensitive in what way?

Reply:
Prone to blowing up if too many instances of the largest string are used. Seeding the RNG prevents that.

Reply:
@hyperbolic2346 did you want a change to the comment here?